adhocteam/Head-Start-TTADP
# Initial basic backend with docker and nodemon

Very basic initial backend with docker and nodemon.

To test it:

- `npm run docker:start`
- open localhost:3000
- modify src/index.js and enter a different message
- reload localhost:3000 - the message should be updated
